core/src/nl/tudelft/watchdog/core/logic/event/eventtypes/TrackingEventType.java

```diff
@@ -6,57 +6,70 @@
  * Enumeration of the different possible debug events that can be fired based on the
  * activities a developer can perform.
  */
-public enum DebugEventType {
+public enum TrackingEventType {
```

Comment:
I think this is completely analogous to our IntervalType enum, and should thus follow the same naming scheme

Reply:
I am  not sure which naming scheme you mean. Could you give a suggestion for a better name?